<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Full_Brain_Parcellation</class>
 <widget class="qMRMLWidget" name="Full_Brain_Parcellation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>620</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>432</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Model:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>T1 : full parcellation w/o prior (Task2010)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T1 : full parcellation with prior (Task2110)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T1 : full parcellation with prior, w/o BET (Task2120)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Channel 1:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Channel 2:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector_channel2">
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector_channel1">
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_2">
     <property name="text">
      <string>Preprocessing</string>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Affine registration:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="brainextrCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Brain extraction:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="affineregCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>Outputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Segmentation</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="outputSelector">
        <property name="toolTip">
         <string>Pick the output to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLSegmentationNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>true</bool>
        </property>
        <property name="removeEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="otherCollapsibleButton">
     <property name="text">
      <string>Other</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>External Python Path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Full_Brain_Parcellation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector_channel1</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Full_Brain_Parcellation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Full_Brain_Parcellation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector_channel2</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
